q2cli/core/config.py

```diff
@@ -0,0 +1,98 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2019, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+import configparser
+
+import click
+
+import q2cli.util
+
+
+class CLIConfig():
+    path = os.path.join(q2cli.util.get_app_dir(), 'cli-colors.theme')
+    VALID_SELECTORS = frozenset(
+        ['option', 'type', 'default_arg', 'command', 'emphasis', 'problem',
+         'error', 'required', 'success'])
+    VALID_STYLINGS = frozenset(
+        ['fg', 'bg', 'bold', 'dim', 'underline', 'blink', 'reverse'])
+    VALID_COLORS = frozenset(
+        ['black', 'red', 'green', 'yellow', 'blue', 'magenta', 'cyan', 'white',
+         'bright_black', 'bright_red', 'bright_green', 'bright_yellow',
+         'bright_blue', 'bright_magenta', 'bright_cyan', 'bright_white'])
+    VALID_BOOLEANS = {'true': True,
+                      'false': False,
+                      't': True,
+                      'f': False}
+
+    def __init__(self):
+        self.styles = self._get_default_styles()
+        try:
+            self.parse_file(self.path)
+        except Exception:
+            if os.path.exists(self.path):
+                os.unlink(self.path)
```

Comment:
We can just reset the theme to default and raise the error through `exit_with_error` without deleting the file. The file is only being deleted from the directory we're parsing it from not wherever they imported it from, but I agree it is unnecessary to delete the file at this point.

Reply:
We can also raise the error and maintain whatever their old settings were

Reply:
This is in __init__, so the damage has already been done... somehow. e.g. maybe they manually edited it instead of installing it, so they skipped verification. Or perhaps they upgraded in-place (how??).

Reply:
right they could theoretically dig into the file I forgot about that. Let me change some things from https://github.com/qiime2/q2cli/pull/217/commits/d9544037711609a91a11eb9db92bee0483b408cd with that in mind

Reply:
@ebolyen what should the error message be if their file fails to parse when it is already in the import directory? Should we suggest they can go to `path` and fix it manually? Or just tell them to `reset-theme`. If something is wrong with that file they probably did fiddle with it manually somehow. I can't think of any other way for it to get messed up outside of some sort of data corruption.